config/routes.rb

```diff
@@ -5,5 +5,7 @@
   # Defines the root path route ("/")
   root "flights#index"
 
-  resources :flights, only: [:index]
+  resources :flights, only: [:index] do
+    post :filter, on: :collection
```

Comment:
1. This should be `get` instead of `post`
2. I don't think you need a route just for filtering, you can use `index` for that

Reply:
Agree, and aligned with the comment: https://github.com/ImMPrada/TOP-flights-booking/pull/4#discussion_r1206111745

So we'll have something `/flights?departure_city_id=481&arrival_city_id=&departure_date=2023-05-29&commit=Search`

Reply:
Done!

Reply:
cool cool!